lumicks/pylake/kymotracker/detail/calibrated_images.py

```diff
@@ -0,0 +1,139 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class CalibratedKymographChannel:
+    """A kymograph image along with its calibration
+
+    Parameters
+    ----------
+    name : str
+        Name of this channel
+    data : array_like
+        2D image data.
+    time_step_ns : float
+        Line time [nanoseconds].
+    pixel_size : float
+        Pixel calibration.
+    """
+
+    def __init__(self, name, data, time_step_ns, pixel_size):
+        self.name = name
+        self.data = data
+        self.time_step_ns = time_step_ns
+        self._pixel_size = pixel_size
```

Comment:
Since we're using them anyway, we could make this a `dataclass` also. Not necessary, but might be good to have consistency for all the classes that just store inputs from `__init__()`

Reply:
I'd like to hold off on this for now. Especially since `_pixel_size` is private for now.